app/participants/knight.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import annotations
+
+
+class Knight:
+    def __init__(self, name: str, power: int, hp: int) -> None:
+        self.name = name
+        self.power = power
+        self.hp = hp
+        self.protection = 0
+
+    def attribute_counter(

```

Comment:
The same comment regarding function naming relates here as well. Rename this method to `apply_attributes`.